docs/UserGuide.md

```diff
@@ -341,22 +342,34 @@ Displays detailed information about a specified person.
 Deletes a specified person from the address book or removes wedding jobs assigned to specified person.
 
 - **Formats**:
-    - **By Index**: `delete INDEX [w/WEDDING_INDEX]...`
-    - **By Name**: `delete NAME [w/WEDDING_INDEX]...`
+    - **By Index**: `delete INDEX`
+    - **By Name**: `delete NAME`
 - **Notes**:
     - **Deleting a person**:
+        - If multiple matches are found when using name-based format, the system will prompt for index.
         - Cannot delete a client who has an active wedding.
           - Error: "Cannot delete this person as they are a client in a wedding. Please delete their wedding first."
-        - If multiple matches are found when using name-based format, the system will prompt for index.
-  - **Removing wedding jobs assigned to a person**:
-      - Can remove multiple weddings jobs a person is assigned to.
-      - Wedding indices must be valid and refer to weddings that person is already assigned to.
+        - Deleting a person will remove them from their assigned weddings.
+
 - **Examples**:
     - `delete 2` deletes the person at index 2.
     - `delete Betsy` deletes Betsy if there's only one match.
-    - `delete 1 w/1` unassigns wedding at index 1 from person at index 1.
-    - `delete Alice w/1 w/2` unassigns Alice from weddings at index 1 and 2 if there's only one match to 'Alice'.
+  
+#### Removing Wedding Jobs Assigned to a Person: `delete`
+
+Removes wedding jobs assigned to specified person.
 
+- **Formats**:
+    - **By Index**: `delete INDEX [w/WEDDING_INDEX]...`
+    - **By Name**: `delete NAME [w/WEDDING_INDEX]...`
+- **Notes**:
+    - **Removing wedding jobs assigned to a person**:
+        - If multiple matches are found when using name-based format, the system will prompt for index.
+        - Can remove multiple weddings jobs a person is assigned to.
+        - Wedding indices must be valid and must refer to weddings that the specified person is already assigned to.
+- **Examples**:
```

Comment:
maybe can add a note saying if no weeding index is provided the contact at index will be deleted?

Reply:
fixedd